index.html

```diff
@@ -85,13 +84,79 @@
 			</ul>
 		</div>
 		<div class="product-details-container">
+			<div class="product-redirect">Back to <a href="#" class="product-selected-category">Woman<a/></div>
+			<h1 class="product-name">The Atelier Tailored Coat</h1>
+			<div class="rating-wrapper">
+			</div>
+			<div class="product-rating-review">
+				<div class="product-counter-review"><!-- it is a dynamic block to display reviews level -->
+				</div>
+				Review(s) <a href="#" class="product-review-button">add review</a>
+			</div>
+			<div class="product-price">&#36;499.00</div>
+			<ul class="product-additional-info">
+				<li class="product-additional-info-availability"><span class="product-additional-info-property">Availability :</span><span class="product-additional-info-value">In stock</span></li>
+				<li class="product-additional-info-availability">Product code :<span class="product-additional-info-value"> &#35;4657</li>
+				<li class="product-additional-info-availability">Tags:
+					<a href="#" class="product-fashion">Fashion</a>
+					<a href="#" class="product-hood">Hood</a>
+					<a href="#" class="product-classic">Classic</a>
+				</li>
+			</ul>
+			<p class="product-description">Sleek, polished, and boasting an impeccably modern fit, this blue, 2-but-
+			ton Lazio suit features a notch lapel, flap pockets, and accompanying flat
+			front trousers—all in pure wool by Vitale Barberis Canonico.
+			<ul>
+				<li>Dark blue suit for a tone-on-tone look</li>
+				<li>Regular fit</li>
+				<li>100% Cotton</li>
+				<li>Free shipping with 4 days delivery</li>
+			</ul>
+			</p>
+			<form method="post" action="product_order.php" name="product-order" class="product-order">
+				<div class="color-selector-wrapper">
+					<label for="color-selector-lable" class="color-selector-label">Color</label>
+					<select class="product-order-color-seletor" name="product-order-color-selector"  id="color-selector-lable">
```

Comment:
1. your id contains "label" word, please remove, it's a selector
2. also, you don't need specific class for select - name and id should be unique - classname is just for styling